ios/Pods/Frameworks/Indy.framework/Headers/IndyAnoncreds.h

```diff
@@ -415,6 +415,17 @@ And is documented in this HIPE:
                  walletHandle:(IndyHandle)walletHandle
                    completion:(void (^)(NSError *error, NSString *outCredID))completion;
 
+/**
+ Deletes credential by given id.
+ @param credId: Identifier by which requested credential is stored in the wallet
+ @param walletHandle Wallet handler (created by IndyWallet::openWalletWithName).
+ @param completion Callback that takes command result as parameter.
+ Returns void
+ */
++ (void)proverDeleteCredentialsWithId:(NSString *)credId
+                         walletHandle:(IndyHandle)walletHandle
+                           completion:(void (^)(NSError * error))completion;
```

Comment:
```suggestion
                           completion:(void (^)(NSError * error, NSString *anonString))completion;
```
You have to match the completion closure which is (Error?, String?) or the iOS will not build

Reply:
Not too familiar with this, but it returns a void so this is correct I think.

Reply:
It returns void but the closure takes in 2 parameters not 1

Reply:
Ah I see what is the issue. The indy-sdk-react-native uses `completionWithString` and it should be `completion`